stonesoup/types/prediction.py

```diff
@@ -75,8 +83,8 @@ def __init__(self, state_vector, covar, timestamp=None,
                  cross_covar=None, *args, **kwargs):
         if(cross_covar is not None
            and cross_covar.shape[1] != state_vector.shape[0]):
-            raise ValueError("cross_covar should have the same number of "
-                             "columns as the number of rows in state_vector")
+            raise ValueError("cross_covar should have the same number of \
+                             columns as the number of rows in state_vector")
```

Comment:
This was better before. This change add loads of white space to error message.

Reply:
Hm, don't remember doing that one. Undone!